gin/sentrygin.go

```diff
@@ -52,9 +126,21 @@ func (h *handler) handle(ctx *gin.Context) {
 		hub = sentry.CurrentHub().Clone()
 	}
 	hub.Scope().SetRequest(ctx.Request)
+
+	transaction := sentry.StartTransaction(
+		ctx, fmt.Sprintf("%v %v", ctx.Request.Method, ctx.FullPath()),
+		sentry.ContinueFromHeaders(h.getTraceIDFromRequest(ctx), h.getBaggageFromRequest(ctx)),
+	)
+	ctx.Writer.Header().Set(traceHeader, transaction.ToSentryTrace())
+	ctx.Writer.Header().Set(baggageHeader, transaction.ToBaggage())
```

Comment:
Can you explain why we need to do this?

Reply:
I’m not sure whether it’s useful to return trace info in HTTP header, like Sentry’ Python SDK works with Flask&FastAPI.

Is there any spec for this use case?